### PR TITLE
[MX-464] Hide manage cards on Eigen

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves **[[MX-464](https://artsyproduct.atlassian.net/browse/MX-427)]**

### Description

For the BNMO flows on iOS, the Manage cards link should be hidden when rendered within the Eigen modal.

### Screenshots
hide_eigen

### CODE REVIEW

**Comment by MounirDhahri** [comment]
sure 🎉

**Comment by ds300** [comment]
I still see the manage cards link 🤔  Might be that my setup is incorrect. Did you test this in the simulator?

**Comment by MounirDhahri** [comment]
My bad, it's fixed now. I found out that `isEigen` flag was broken on the client side and it was breaking other functionalities in Eigen such as [Zoom on order](https://artsyproduct.atlassian.net/browse/PURCHASE-464). It's fixed now.
hide_eigen

File: src/v2/Artsy/SystemContext.tsx [hunk 1]
```diff
@@ -103,6 +104,7 @@ export const SystemContextProvider: SFC<SystemContextProps> = ({
     relayEnvironment,
     user,
     setUser,
+    isEigen: sd.EIGEN,
```
Comment (MounirDhahri) [inline src/v2/Artsy/SystemContext.tsx:107]: isEigen is already a prop in the same file but wasn't being set
https://github.com/artsy/force/blob/741dec64e9828067b3f1fce45dfa664862a868c9/src/v2/Artsy/SystemContext.tsx#L49

**Comment by ds300** [comment]
@MounirDhahri looks like there's a couple of failing tests?

**Comment by ds300** [comment]
nice one 🙏

**Pull request merged**